client/lib/products-values/index.js

```diff
@@ -211,6 +211,15 @@ export function isJetpackProduct( product ) {
 	return includes( jetpackProducts, product.product_slug );
 }
 
+export function isJetpackBackup( product ) {
```

Comment:
Why not just use the [`isJetpackProduct()` function](https://github.com/Automattic/wp-calypso/pull/37448/files#diff-a33b3012cc6794a297b3c589a2df0b3cR205) that we already have a few lines above?

Reply:
I guess in the long run having that function makes sense - this can be used to specify an icon for the backup products, and that icon should apply for them only, and not for other Jetpack products.

However, right now it's an over-engineering, as we only have the Backup product - let's just use `isJetpackProduct` as @delawski suggested.

Reply:
I think it makes sense to have a new function `isJetpackBackup` because we want to make the cloud icon only ever appear for backups (ever). We will need to come up with a different icon for other cases.

Reply:
But this function is currently the same as the other one? If we need it anyway, could we at least 
reuse the existing function inside this new one, and add a comment to explain the purpose?